lib/roger_ui/jobs.ex

```diff
@@ -1,6 +1,53 @@
-defmodule RogerUi.Jobs do
+defmodule RogerUI.Jobs do
   @moduledoc """
-  Generate Jobs list from Roger.Info.running_jobs()
+  Generates a Jobs list from Roger.Info.running_jobs() function.
+  Resulting Jobs are linked to a partition name:
+
+  Given a nested data structure, where each element contents nested items:
+  [
+      "watcher@127.0.0.1": %{
+        "roger_demo_partition" => [
+          %Roger.Job{
+            args: %{"country" => "Venezuela", "email" => "janedoe@gmail.com", "name" => "Jane Doe", "number_of_pets" => 966},
+            execution_key: nil, id: "16ovjr39jvijf4kgrlqe2ib2aadojhid", module: RogerDemo.Job.CreateUpdateUser,
+            queue_key: nil, queued_at: 1517697586453, retry_count: 0, started_at: 1517697682999
+          }
+  ]]
+
+  return a sorted Map with the following structure:
+  [
+  %Roger.Job{
+    args: %{
+      "country" => "Venezuela",
+      "email" => "janedoe@gmail.com",
+      "name" => "Jane Doe",
+      "number_of_pets" => 966
+    },
+    execution_key: nil,
+    id: "16ovjr39jvijf4kgrlqe2ib2aadojhid",
+    module: RogerDemo.Job.CreateUpdateUser,
+    queue_key: nil,
+    queued_at: 1517697586453,
+    retry_count: 0,
+    started_at: 1517697682999
+  }]
+  """
+
+  @doc """
+  Takes a Keyword list with nodes, partitions and jobs like this:
+
+  [
```

Comment:
Can you put code examples in markdown code blocks?

Reply:
done